setup.py

```diff
@@ -51,6 +51,12 @@ def read_version_from_package_xml():
     return version.text
 
 
+def ament_lint_url(subpackage):
+    repo_url = 'https://github.com/ament/ament_lint.git'
+    return 'git+{repo}@{ver}#egg={pkg}-{ver}&subdirectory={pkg}'.format(
+        repo=repo_url, ver='0.8.1', pkg=subpackage)
```

Comment:
Maybe extract 0.8.1 to a named constant and add a couple lines of when/how we should bump this? Is it ok to just whatever latest tag is available?

Reply:
k - i'll add a note about that. whatever latest tag is available should be perfectly fine, whenever we feel like bumping it